src/machines/java/com/enderio/machines/data/recipes/MachineRecipeProvider.java

```diff
@@ -25,6 +26,62 @@ public MachineRecipeProvider(PackOutput pOutput) {
 
     @Override
     protected void buildRecipes(Consumer<FinishedRecipe> finishedRecipeConsumer) {
+        ShapedRecipeBuilder
+            .shaped(RecipeCategory.MISC, MachineBlocks.CAPACITOR_BANKS.get(CapacitorTier.BASIC).get())
+            .define('I', Tags.Items.INGOTS_IRON)
+            .define('C', EIOItems.BASIC_CAPACITOR.get())
+            .define('R', Tags.Items.STORAGE_BLOCKS_REDSTONE)
+            .pattern("ICI")
+            .pattern("CRC")
+            .pattern("ICI")
+            .unlockedBy("has_ingredient", InventoryChangeTrigger.TriggerInstance.hasItems(ItemPredicate.Builder.item().of(EIOItems.BASIC_CAPACITOR).build()))
+            .save(finishedRecipeConsumer);
+
+        ShapedRecipeBuilder
+            .shaped(RecipeCategory.MISC, MachineBlocks.CAPACITOR_BANKS.get(CapacitorTier.ADVANCED).get())
+            .define('A', EIOItems.COPPER_ALLOY_INGOT.get())
+            .define('C', EIOItems.DOUBLE_LAYER_CAPACITOR.get())
+            .define('R', Tags.Items.STORAGE_BLOCKS_REDSTONE)
+            .pattern("ACA")
+            .pattern("CRC")
+            .pattern("ACA")
+            .unlockedBy("has_ingredient", InventoryChangeTrigger.TriggerInstance.hasItems(ItemPredicate.Builder.item().of(EIOItems.BASIC_CAPACITOR).build()))
+            .save(finishedRecipeConsumer, "advanced_capacitor_bank");
```

Comment:
Instead of using a string, put this string through EnderIO.loc(...) to make sure that its in the enderio: namespace instead of minecraft: (see this PR's file list to see what I mean). You also don't need it in this location, only in places where the second recipe's automatic name would conflict (so the upgrade recipes)

Reply:
Resolved!